ArduSub/AP_Arming_Sub.cpp

```diff
@@ -48,7 +55,7 @@ bool AP_Arming_Sub::pre_arm_checks(bool display_failure)
     }
     // don't allow arming unless there is a disarm button configured
     if (!has_disarm_function()) {
-        check_failed(display_failure, "Must assign a disarm or arm_toggle button");
+        check_failed(display_failure, "Must assign a disarm or arm_toggle button or disarm aux function");
```

Comment:
we could make this shorter, something like, "set RCx_OPTION = yy (Disarm)"

Reply:
that is not the correct message.....it could be "Needs a disarm switch or button setup"...I will make another PR if you are really fussed...